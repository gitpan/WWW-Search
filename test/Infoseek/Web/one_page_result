http://www.infoseek.com/Titles?rf=0&qt=Martin+Thurn+AND+Star+Wars&col=WW&nh=25&st=0&
