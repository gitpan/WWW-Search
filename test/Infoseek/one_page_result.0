<html><head><base target="_top"><meta name="IS" content="ok_spider3"><title>Search results for: Martin Thurn AND Star Wars</Title></head><body bgcolor="#FFFFFF" link="#0000CC" vlink="#990099">
<table width="100%" cellpadding="0" cellspacing="0" border="0">
<tr><td rowspan="2" width="1%"><a href="/Home/Home.html?svx=search_header&nh=25"><img src="/images/tmpl/search_go.gif" width="120" height="38" border="0" hspace="0" vspace="0"></td>
<td height="23" width="100%">&nbsp;</td></tr>
<tr><td bgcolor="#000000" height="15"><spacer type="block" width="1" height="15"></td></tr>
</table><font face="Helvetica,Arial" size="2"><a href="/Home/Home.html?svx=search_breadcrumb&rf=0&qt=Martin+Thurn+AND+Star+Wars&col=WW&nh=25&st=0">GO</a> &gt; <b>Martin Thurn AND Star Wars</b> <br></font><table width="100%" border="0" cellpadding="0" cellspacing="0" bgcolor="#DEDECA"><tr><td bgcolor="#000000" colspan="3" height="1"><spacer type="block" width="1" height="1"></td></tr><tr><td bgcolor="#999999" colspan="3" height="1"><spacer type="block" width="1" height="1"></td></tr><tr><td bgcolor="#DEDECA" colspan="3">&nbsp;</td></tr><tr><td width="40%" rowspan="3">&nbsp;</td><td nowrap><font face="Helvetica,Arial" size="3"><form action="/Titles" method="get" name="WW_Search"><b>Search</b></font><font face="Verdana,Helvetica,Arial" size="1">&nbsp;<input type="radio" value="WW" name="col" checked><b>New</b> search&nbsp;<input type="radio" value="WW,cat_RES" name="col">Search <b>within</b> results</font><br><img src="/images/clear.gif" width="1" height="1"></td><td width="40%" rowspan="3">&nbsp;</td></tr><tr><td valign="top" nowrap><input type="text" name="qt" value="" size="35" maxlength="511">&nbsp;<INPUT TYPE=hidden NAME="sv" VALUE="IS">
<INPUT TYPE=hidden NAME="lk" VALUE="noframes">
<INPUT TYPE=hidden NAME="nh" VALUE="25">
<input type=hidden name="cc" value="WW">
<input type=hidden name="oq" value="Martin Thurn AND Star Wars">
<input type="submit" value="Find"></td></tr><tr><td nowrap><font face="Verdana,Helvetica,Arial" size="1"><a href="/find?pg=advanced_www.html&ud9=advanced_www&nh=25">Search&nbsp;options</a>&nbsp;|&nbsp;<a href="/Help?pg=SearchTips.html&nh=25">How to search</a>&nbsp;|&nbsp;</font><br><img src="/images/clear.gif" height="4" width="1"></td></tr></form><!-- End Search Box --><tr><td colspan="3" height="8"><spacer type="block" width="1" height="8"></td></tr></table><table border="0" cellpadding="0" cellspacing="0" width="100%"><tr><td bgcolor="#000000" height="1"><spacer type="block" width="1" height="1"></td></tr></table>
<img src="/images/clear.gif" border="0" width="1" height="6"><br>
<center><table width="100%" cellpadding="0" cellspacing="0" border="0"><tr><td align="center" valign="top"><table align="left" width="121" height="60" cellpadding="1" cellspacing="0" border="0">
<tr><td bgcolor="#000000" valign="top">
<table width="119" height="58" cellpadding="0" cellspacing="0" border="0">
<tr><td bgcolor="#FFFFFF" align="center" valign="bottom" width="119" NOWRAP><font face="Arial,Helvetica" size="-2">Books at</font></td></tr>
<tr><td bgcolor="#FFFFFF" align="center" valign="bottom" width="119" NOWRAP><font face="Verdana,Arial,Helvetica" size="-1"><b><a href="/redirect?sv=IS&lk=noframes&ak=BORD&rl=LINK-C0-standard-flash-borders&_nr=1&rd=http://www.borders.com/cgi-bin/partner?PARTNER_ID=101010097&Url=http://www.borders.com/infoseek/section_10000.html">Borders.com</a></b></font></td></tr>
<tr><td bgcolor="#FFFFFF" align="center" valign="bottom" width="119" NOWRAP><font face="Arial,Helvetica" size="-2">In stock, Fast delivery.</font></td></tr>
<tr><td bgcolor="#FFFFFF" align="center" valign="bottom" width="119" NOWRAP><a href="/redirect?sv=IS&lk=noframes&ak=BORD&rl=LINK-C0-standard-flash-borders&_nr=1&rd=http://www.borders.com/cgi-bin/partner?PARTNER_ID=101010097&Url=http://www.borders.com/infoseek/section_10000.html"><img src="/images/partners/borders/031999borders_logo.gif" width="119" height="16" border="0"></a><br></td></tr>
</table>
</td></tr>
</table>
</td><td>&nbsp;&nbsp;</td><td valign="top" align="center" colspan="3"><a href="http://www.infoseek.com/redirect?sv=IS&lk=&tid=&ak=IMA0-2000-1001-GEN&rd=http://www.imall.com/holiday/" target="_top"><img border=1 width=468 height=60 src="/ads/IMA01001.gif" alt="[iMall]"><br><FONT FACE="verdana,arial,helvetica" SIZE="1"><b>Click here</b></FONT></a></td></tr></table></center><br>
<table width="100%" border="0" cellpadding="0" cellspacing="0" bgcolor="#003366">
<tr><td valign="middle" align="left" nowrap colspan="3"><font face="Helvetica,Arial" size="3" color="#FFFFFF"><a name="search">&nbsp;<b>Web search results</b>&nbsp;&nbsp;&nbsp;&nbsp;<font face="Helvetica,Arial" size="2">1 - 19 of <b>19</b> results most relevant to <b>Martin Thurn AND Star Wars</b> </font>&nbsp;</a></font></td>
<td height="18" valign="middle" align="right" nowrap  colspan="1">&nbsp;
</td></tr>
<tr><td valign="middle" align="left" nowrap colspan="4" bgcolor="#000000"><img src="/images/clear.gif" border="0" width="100%" height="1"></td>
</table>
<spacer type="block" width="1" height="2"><!-- Begin RHS -->
<table cellpadding="1" cellspacing="0" border="0" width="150" align="right" bgcolor="#FFFFFF"><tr><td height="4"><spacer type="block" width="1" height="4"></td></tr><tr><td></td></tr><tr><td><table cellpadding="1" cellspacing="0" border="0" width="150" align="right" bgcolor="#000000"><tr><td><table cellpadding="6" cellspacing="0" border="0" width="148" align="right" bgcolor="#FFFFFF"><tr><td><center><font size="2" face="Verdana,Helvetica,Arial"><b>Buy the book</b></font></center><br><!-- Infoseek cut-->
<FONT FACE="Arial,Helvetica" SIZE="2">Win an<A HREF="/redirect?sv=IS&lk=&ak=BORD&rl=LINK_C99999_UNMAPPED_RESULTS_GO&rd=http://www.borders.com/cgi-bin/part?PID=122010891&PKIND=aa&PAGE=http%3A%2F%2Fsearch%2Eborders%2Ecom%2Fpromo%2Fimac%2Findex%2Excv"> iMac</A>!
<BR><A HREF="/redirect?sv=IS&lk=&ak=BORD&rl=LINK_C99999_UNMAPPED_RESULTS_GO&rd=http://www.borders.com/cgi-bin/part?PID=122010892&PKIND=aa&PAGE=http%3A%2F%2Fsearch%2Eborders%2Ecom%2Fboutiques%2Fstarwars%2Findex%2Excv">Star Wars</A> must-haves!
<br>Find the perfect <a href="/redirect?sv=IS&lk=&ak=BORD&rl=LINK_C99999_UNMAPPED_RESULTS_GO&rd=http://www.borders.com/cgi-bin/part?PID=122010894&PKIND=aa&PAGE=http%3A%2F%2Fsearch%2Eborders%2Ecom%2Fgc%2Findex%2Excv">Graduation Gift</a>.
<BR><a href="/redirect?sv=IS&lk=&ak=BORD&rl=LINK_C99999_UNMAPPED_RESULTS_GO&rd=http://www.borders.com/cgi-bin/part?PID=122010893&PKIND=aa&PAGE=http%3A%2F%2Fsearch%2Eborders%2Ecom%2Ffcgi%2Dbin%2Fdb2www%2Fbgilistitems%2Ed2w%2Freport%3FListRef%3D2206">Gay Pride Month</A> Books!</FONT>
<!-- Infoseek cut--></td></tr></table></td></tr></table></td></tr></table>
<!-- End RHS -->
<font face="Helvetica,Arial" size="2"><a href="http://infoseek.go.com/Titles?rf=0&qt=Martin+Thurn+AND+Star+Wars&col=WW&nh=20&st=0&&sf=1&ud4=1">Hide summaries</a>&nbsp;&nbsp;|&nbsp;&nbsp; <a href="http://infoseek.go.com/Titles?rf=0&qt=Martin+Thurn+AND+Star+Wars&col=WW&nh=25&st=0&&sf=3">Sort by date</a>&nbsp;&nbsp;|&nbsp;&nbsp;<a href="http://infoseek.go.com/Titles?rf=0&qt=Martin+Thurn+AND+Star+Wars&col=WW&nh=25&st=0&&sf=0">Group results</a></font><br><br><font face="Helvetica,Arial" size="2"><b><a href="http://www.cs.washington.edu/homes/lopez/faq.html">REC.ARTS.SF.STARWARS.COLLECTING FAQ ***** Last updated 4-1-96 ***** Coordinators: Gus Lopez ...</a></b><br>REC.ARTS.SF.STARWARS.COLLECTING FAQ ***** Last updated 4-1-96 ***** Coordinators: Gus Lopez Chris Nichols Authors: Dave Halsted Gus Lopez Chris Nichols ...<br><font face="Helvetica,Arial" size="2">78% &nbsp;<b>Date:  8 May 1996</b>, &nbsp;Size 25.2K, &nbsp;http://www.cs.washington.edu/homes/lopez/faq.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=73464968934424513&fs=http%3A//www.cs.washington.edu/homes/lopez/faq.html&svx=find_similar">Find similar pages</a><p><b><a href="http://force.stwing.upenn.edu/starwars/collectfaq.txt">REC.ARTS.SF.STARWARS.COLLECTING FAQ ---- Coordinators: Gus Lopez (lopez@cs.washington.edu) ...</a></b><br>REC.ARTS.SF.STARWARS.COLLECTING FAQ ---- Coordinators: Gus Lopez Chris Nichols Authors: Dave Halsted Gus Lopez Chris Nichols <b style="background:#FFFF99">Martin Thurn</b> Contributors: Wolfgang ...<br><font face="Helvetica,Arial" size="2">77% &nbsp;<b>Date: 25 Jun 1995</b>, &nbsp;Size 22.5K, &nbsp;http://force.stwing.upenn.edu/starwars/collectfaq.txt&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751048996118&fs=http%3A//force.stwing.upenn.edu/starwars/collectfaq.txt&svx=find_similar">Find similar pages</a><p><b><a href="http://www.pitt.edu/~thurn/"><b style="background:#FFFF99">Martin Thurn's</b> Index Page</a></b><br>I am gainfully employed at TASC. I have a family including 3 beautiful children. I am a co-habitant with two immaculate cats. I am the editor of , a bimonthly newsletter by <b style="background:#FFFF99">Star Wars</b> collectors, for <b style="background:#FFFF99">Star Wars</b> ...<br><font face="Helvetica,Arial" size="2">70% &nbsp;<b>Date:  9 Mar 1998</b>, &nbsp;Size 1.0K, &nbsp;http://www.pitt.edu/~thurn/&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=73464968939049189&fs=http%3A//www.pitt.edu/%7Ethurn/&svx=find_similar">Find similar pages</a><p><b><a href="http://www.pitt.edu/~thurn/SWB/opeechee.html">The <b style="background:#FFFF99">Star Wars</b> Collector's Bible (O-Pee-Chee)</a></b><br><br><font face="Helvetica,Arial" size="2">65% &nbsp;<b>Date:  3 Feb 1999</b>, &nbsp;Size 4.5K, &nbsp;http://www.pitt.edu/~thurn/SWB/opeechee.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751059999033&fs=http%3A//www.pitt.edu/%7Ethurn/SWB/opeechee.html&svx=find_similar">Find similar pages</a><p><b><a href="http://www.cs.washington.edu/homes/lopez/info.html">Information on the <b style="background:#FFFF99">Star Wars</b> Collectors Archive</a></b><br>The <b style="background:#FFFF99">Star Wars</b> Collectors Archive  Created by  Gus Lopez  (lopez@cs.washington.edu).  Much of the information contained here is researched and written by Chris Georgoulias (stun ...<br><font face="Helvetica,Arial" size="2">63% &nbsp;<b>Date: 14 Aug 1996</b>, &nbsp;Size 5.2K, &nbsp;http://www.cs.washington.edu/homes/lopez/info.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751038304868&fs=http%3A//www.cs.washington.edu/homes/lopez/info.html&svx=find_similar">Find similar pages</a><p><b><a href="http://www.suba.com/~tydirium/bhlinks.html">Blue Harvest Magazine</a></b><br><b style="background:#FFFF99">Star Wars</b> Fanzine<br><font face="Helvetica,Arial" size="2">61% &nbsp;<b>Date: 12 Oct 1998</b>, &nbsp;Size 4.1K, &nbsp;http://www.suba.com/~tydirium/bhlinks.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=73464968935985794&fs=http%3A//www.suba.com/%7Etydirium/bhlinks.html&svx=find_similar">Find similar pages</a><p><b><a href="http://www.stwing.upenn.edu/starwars/collectfaq.txt">---- REC.ARTS.SF.STARWARS.COLLECTING FAQ ---- Coordinators: Gus Lopez (lopez@cs.w ...</a></b><br><br><font face="Helvetica,Arial" size="2">60% &nbsp;<b>Date: 25 Jun 1995</b>, &nbsp;Size 22.5K, &nbsp;http://www.stwing.upenn.edu/starwars/collectfaq.txt&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751060142198&fs=http%3A//www.stwing.upenn.edu/starwars/collectfaq.txt&svx=find_similar">Find similar pages</a><p><b><a href="http://www.pitt.edu/~thurn/SWB/">The <b style="background:#FFFF99">Star Wars</b> Collector's Bible (Introduction)</a></b><br>The <b style="background:#FFFF99">Star Wars</b> Collector's Bible is a detailed hypertext list of U.S.-released <b style="background:#FFFF99">Star Wars</b>-related merchandise. The SW Bible is organized alphabetically . Within each company listing, items are in order ...<br><font face="Helvetica,Arial" size="2">59% &nbsp;<b>Date: 19 Dec 1998</b>, &nbsp;Size 9.0K, &nbsp;http://www.pitt.edu/~thurn/SWB/&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751051593649&fs=http%3A//www.pitt.edu/%7Ethurn/SWB/&svx=find_similar">Find similar pages</a><p><b><a href="http://www.cs.washington.edu/homes/lopez/textfiles.html">Charts, Tables, and Text Files</a></b><br>Text Files Maintained by Ever wonder which weapon goes with which figure? Well, that's the kind of information you can find right here. Any beginning collector should glance at some of these files since they answer many questions that are likely to come up. More experienced collectors might be ...<br><font face="Helvetica,Arial" size="2">59% &nbsp;<b>Date:  3 Dec 1996</b>, &nbsp;Size 2.9K, &nbsp;http://www.cs.washington.edu/homes/lopez/textfiles.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=73464968930724861&fs=http%3A//www.cs.washington.edu/homes/lopez/textfiles.html&svx=find_similar">Find similar pages</a><p><b><a href="http://pages.map.com/starwars/faq.html"><b style="background:#FFFF99">Star Wars</b> Collecting FAQ</a></b><br>Last Updated: 9-23-97 Below is a list of Frequently Asked Questions in regard to <b style="background:#FFFF99">Star Wars</b> collecting, mostly having to do with Kenner toys. The list is always a work in progress, if you have any suggestions ...<br><font face="Helvetica,Arial" size="2">55% &nbsp;<b>Date: 24 Sep 1997</b>, &nbsp;Size 39.5K, &nbsp;http://pages.map.com/starwars/faq.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751047352138&fs=http%3A//pages.map.com/starwars/faq.html&svx=find_similar">Find similar pages</a><p><b><a href="http://www.pitt.edu/~thurn/SWC/">The <b style="background:#FFFF99">Star Wars</b> Collector</a></b><br>INTRODUCTION The <b style="background:#FFFF99">Star Wars</b> Collector is a magazine by collectors, for collectors, about all aspects of collecting all <b style="background:#FFFF99">Star Wars</b>-related items of all time periods. Each issue of The <b style="background:#FFFF99">Star Wars</b> Collector is filled with ...<br><font face="Helvetica,Arial" size="2">55% &nbsp;<b>Date:  1 Dec 1998</b>, &nbsp;Size 7.6K, &nbsp;http://www.pitt.edu/~thurn/SWC/&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=73464968939730562&fs=http%3A//www.pitt.edu/%7Ethurn/SWC/&svx=find_similar">Find similar pages</a><p><b><a href="http://www.cs.washington.edu/homes/lopez/textf/SWfigures.html">ACTION FIGURES SMALL ACTION FIGURES, Kenner, 1978-1985 Came blister packaged on color photo ...</a></b><br>ACTION FIGURES SMALL ACTION FIGURES, Kenner, 1978-1985 Came blister packaged on color photo headers. They were released in eight series , new figures added with each film. Older figures were not dropped from production but ...<br><font face="Helvetica,Arial" size="2">45% &nbsp;<b>Date: 14 Apr 1996</b>, &nbsp;Size 19.0K, &nbsp;http://www.cs.washington.edu/homes/lopez/textf/SWfigures.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751038088301&fs=http%3A//www.cs.washington.edu/homes/lopez/textf/SWfigures.html&svx=find_similar">Find similar pages</a><p><b><a href="http://www.pitt.edu/~thurn/SWB/tpromos.html">The <b style="background:#FFFF99">Star Wars</b> Collector's Bible (Topps promo cards)</a></b><br><br><font face="Helvetica,Arial" size="2">40% &nbsp;<b>Date:  3 Feb 1999</b>, &nbsp;Size 28.4K, &nbsp;http://www.pitt.edu/~thurn/SWB/tpromos.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=73464968945513853&fs=http%3A//www.pitt.edu/%7Ethurn/SWB/tpromos.html&svx=find_similar">Find similar pages</a><p><b><a href="http://www.cs.washington.edu/homes/lopez/textf/all-cards.html"><b style="background:#FFFF99">STAR WARS</b> CARDED FIGURE VARIATIONS ************************************ (revised 6-5-95) ...</a></b><br><b style="background:#FFFF99">STAR WARS</b> CARDED FIGURE VARIATIONS ************************************ NOTES: ------ 1) This is an attempt to assemble the most complete and accurate list of card variations as they were sold for the U.S. market. 2) Figures are ...<br><font face="Helvetica,Arial" size="2">36% &nbsp;<b>Date:  7 Jun 1995</b>, &nbsp;Size 26.5K, &nbsp;http://www.cs.washington.edu/homes/lopez/textf/all-cards.html&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751041286269&fs=http%3A//www.cs.washington.edu/homes/lopez/textf/all-cards.html&svx=find_similar">Find similar pages</a><p><b><a href="http://www.stwing.upenn.edu/starwars/collect/vartions.txt">LOOSE <b style="background:#FFFF99">STAR WARS</b> FIGURE VARIATIONS **** 1. I've left prototypes off the list, except in the ...</a></b><br>, so I'm not advocating searching out all these variations. My goal is to generate the definitive list of loose figure variations, including major differences in sculpting, painting, foreign releases, and accessories. 3. I have ...<br><font face="Helvetica,Arial" size="2">36% &nbsp;<b>Date: 13 Jan 1995</b>, &nbsp;Size 15.9K, &nbsp;http://www.stwing.upenn.edu/starwars/collect/vartions.txt&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=73464968929897453&fs=http%3A//www.stwing.upenn.edu/starwars/collect/vartions.txt&svx=find_similar">Find similar pages</a><p><b><a href="http://www.pitt.edu/%7Ethurn/SWB/">The <b style="background:#FFFF99">Star Wars</b> Collector's Bible (Introduction)</a></b><br>The <b style="background:#FFFF99">Star Wars</b> Collector's Bible (hereafter The SW Bible or SWB ) is a detailed hypertext list of U.S.-released <b style="background:#FFFF99">Star Wars</b>-related merchandise. NEW! Search! Search SWB for: Organization and Access Methods by ...<br><font face="Helvetica,Arial" size="2">28% &nbsp;<b>Date: 11 May 1999</b>, &nbsp;Size 9.9K, &nbsp;http://www.pitt.edu/%7Ethurn/SWB/&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=260082878481765475&fs=http%3A//www.pitt.edu/%257Ethurn/SWB/&svx=find_similar">Find similar pages</a><p><b><a href="http://www.pitt.edu/%7Ethurn/SWC/">The <b style="background:#FFFF99">Star Wars</b> Collector</a></b><br>is a magazine by collectors, for collectors, about all aspects of collecting all <b style="background:#FFFF99">Star Wars</b>-related items of all time periods.  Each issue of  The <b style="background:#FFFF99">Star Wars</b> Collector  is filled with useful and informative articles ...<br><font face="Helvetica,Arial" size="2">28% &nbsp;<b>Date: 23 Mar 1998</b>, &nbsp;Size 7.5K, &nbsp;http://www.pitt.edu/%7Ethurn/SWC/&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=289637751038961377&fs=http%3A//www.pitt.edu/%257Ethurn/SWC/&svx=find_similar">Find similar pages</a><p><b><a href="http://www.myhomepage.net/~sharke/fan18.htm">FANDOM ORGANIZATIONS</a></b><br>DISCLAIMER: ALL INFORMATION CONCERNING FANDOM ACTOR FAN CLUB(S) IS SUBJECT TO CHANGE WITHOUT NOTICE BY CLUB ORGANIZERS. IF YOU HAVE ANY QUESTIONS CONCERNING THE INFORMATION ON THESE CLUBS, PLEASE CONTACT THEM ...<br><font face="Helvetica,Arial" size="2">28% &nbsp;<b>Date: 25 Aug 1998</b>, &nbsp;Size 4.9K, &nbsp;http://www.myhomepage.net/~sharke/fan18.htm&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=73464968925795859&fs=http%3A//www.myhomepage.net/%7Esharke/fan18.htm&svx=find_similar">Find similar pages</a><p><b><a href="http://www.stwing.upenn.edu/starwars/collect/swfigrs.txt">ACTION FIGURES  Small Action Figures, Kenner, 1978-1985, came blister ...</a></b><br>ACTION FIGURES  Small Action Figures, Kenner, 1978-1985, came blister packaged on color photo headers.  They were released in eight series (I-VIII), new figures added ...<br><font face="Helvetica,Arial" size="2">22% &nbsp;<b>Date: 27 Apr 1994</b>, &nbsp;Size 14.4K, &nbsp;http://www.stwing.upenn.edu/starwars/collect/swfigrs.txt&nbsp;</font><br><a href="http://infoseek.go.com/Titles?rf=0&col=WW&nh=25&&sf=1&cat=RES&fsd=73464968921845748&fs=http%3A//www.stwing.upenn.edu/starwars/collect/swfigrs.txt&svx=find_similar">Find similar pages</a></font><br clear=all><br clear=all><font face="Helvetica,Arial" size="2"></font><br><br><center><p><center><a href="http://www.infoseek.com/redirect?sv=IS&lk=&tid=&ak=IMA0-2000-1001-GEN&rd=http://www.imall.com/holiday/" target="_top"><img border=1 width=468 height=60 src="/ads/IMA01001.gif" alt="[iMall]"><br><FONT FACE="verdana,arial,helvetica" SIZE="1"><b>Click here</b></FONT></a></center><p><center>
<table cellpadding="0" cellspacing="0" border="0" width="100%"><tr><td bgcolor="#DEDECA">&nbsp;</td></tr></table><p><table border="0" cellpadding="0" cellspacing="0" width="100%"><tr><td rowspan="4" width="35%" align="center" valign="top"><font size="2" face="Verdana,Helvetica,Arial"><a href="/Titles?nh=25&tv=brag&qt=Martin+Thurn+AND+Star+Wars&col=WW&sv=IS&svx=brag_page"><img src="/images/misc/powered_by_stack.gif" border="0" alt="Powered by Infoseek"><br>What makes us better</a></font></td><tr><td width="40%" rowspan="3">&nbsp;</td><td nowrap><font face="Helvetica,Arial" size="3"><form action="/Titles" method="get" name="WW_Search"><b>Search</b></font><font face="Verdana,Helvetica,Arial" size="1">&nbsp;<input type="radio" value="WW" name="col" checked><b>New</b> search&nbsp;<input type="radio" value="WW,cat_RES" name="col">Search <b>within</b> results</font><br><img src="/images/clear.gif" width="1" height="1"></td><td width="40%" rowspan="3">&nbsp;</td></tr><tr><td valign="top" nowrap><input type="text" name="qt" value="" size="35" maxlength="511">&nbsp;<INPUT TYPE=hidden NAME="sv" VALUE="IS">
<INPUT TYPE=hidden NAME="lk" VALUE="noframes">
<INPUT TYPE=hidden NAME="nh" VALUE="25">
<input type=hidden name="cc" value="WW">
<input type=hidden name="oq" value="Martin Thurn AND Star Wars">
<input type="submit" value="Find"></td></tr><tr><td nowrap><font face="Verdana,Helvetica,Arial" size="1"><a href="/find?pg=advanced_www.html&ud9=advanced_www&nh=25">Search&nbsp;options</a>&nbsp;|&nbsp;<a href="/Help?pg=SearchTips.html&nh=25">How to search</a>&nbsp;|&nbsp;</font><br><img src="/images/clear.gif" height="4" width="1"></td></tr></form><!-- End Search Box --></table><br><br><center><font size="2" face="Verdana,Helvetica,Arial">If there is a site you would like listed, click <a href="/AddUrl?pg=SubmitUrl.html&nh=25&svx=search_footer"><b>add URL</b></a>.<hr><br><center><b>GO Network partners:</b>&nbsp;&nbsp;<a href="http://abc.go.com/"><b>ABC.com</b></a>&nbsp;&nbsp;<a href="http://abcnews.go.com"><b>ABCNEWS.com</b></a>&nbsp;&nbsp;<a href="http://disney.go.com"><b>Disney.com</b></a>&nbsp;&nbsp;<a href="http://espn.go.com"><b>ESPN.com</b></a>&nbsp;&nbsp;<a href="http://family.go.com"><b>Family.com</b></a>&nbsp;&nbsp;<a href="http://mrshowbiz.go.com"><b>Mr. Showbiz</b></a><br><br><a href="/doc?pg=copyright.html">Copyright</a> &copy; 1999 Infoseek Corporation.  All rights reserved.&nbsp;&nbsp;&nbsp;&nbsp;<a href="mailto:comments@infoseek.com">Contact us</a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://info.go.com/doc/policy/disclaimer.html">Legal Notices</a>&nbsp;&nbsp;&nbsp;&nbsp;<a href="http://info.go.com/doc/policy/privacy.html">Privacy policy</a></font><br><br><br></center></body></html><pre>









</pre>
</body>
</html>
