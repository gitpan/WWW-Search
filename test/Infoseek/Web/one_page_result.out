http://www.cs.washington.edu/homes/lopez/textf/all-cards.html
http://www.pitt.edu/%7Ethurn/SWB/
http://www.pitt.edu/%7Ethurn/SWC/
http://www.toysrgus.com/images-bootleg.html
http://www.cs.washington.edu/homes/lopez/info.html
http://www.cs.washington.edu/homes/lopez/textfiles.html
http://www.stwing.upenn.edu/starwars/collect/vartions.txt
http://www.stwing.upenn.edu/starwars/collect/swfigrs.txt
http://www.phoaks.com/phoaks2/newsgroups/rec/toys/action-figures/resources0.html
http://www.cs.washington.edu/homes/lopez/textf/SWfigures.html
http://www.freelaunch.com/
